.cirrus.yml

```diff
@@ -51,7 +51,6 @@ task:
         13.0*) SKIP_TESTS='!1242 !1243 !2002 !2003';;
         12.1*) SKIP_TESTS='!1242 !1243 !2002 !2003';;
         11.3*) SKIP_TESTS='!504 !1242 !1243 !2002 !2003';;
```

Comment:
You could turn those ! into ~ while you are at it.

Reply:
But why? We don't have anything on cirrus that shows/cares about ignore tests.

Reply:
I think the ignore feature is not only for the reporting stuff, but also to see the error output in the CI logs.